src/domains/ecommerce-app/04_apim_ecommerce.tf

```diff
@@ -658,3 +658,21 @@ module "apim_ecommerce_user_stats_service_api_v1" {
     hostname = local.ecommerce_hostname
   })
 }
+
+#################
+## NAMED VALUE ##
+#################
+data "azurerm_key_vault_secret" "ecommerce_dev_sendpaymentresult_subscription_key" {
+  name         = "ecommerce-dev-sendpaymentresult-subscription-key"
```

Comment:
add count in order to avoid fetch data in dev and uat

Reply:
Done here 6b6594f74a8d59fcf893d5b4f9bf2782fdb48290